x-pack/plugins/security_solution/server/lib/telemetry/tasks/endpoint.ts

```diff
@@ -188,7 +190,38 @@ export function createTelemetryEndpointTaskConfig(maxTelemetryBatch: number) {
           )
         : new Map<string, EndpointPolicyResponseDocument>();
 
-      /** STAGE 4 - Create the telemetry log records
+      /** STAGE 4 - Fetch Endpoint Agent Metadata
```

Comment:
I always thought calling these entire documents "EP Metadata" would come back to bite us 😆